src/cpp/src/visual_language/inputs_embedder.cpp

```diff
@@ -239,6 +239,161 @@ std::pair<ov::Tensor, ov::Tensor> InputsEmbedder::IInputsEmbedder::get_inputs_em
 
 bool InputsEmbedder::IInputsEmbedder::has_token_type_ids() const { return false; }
 
+// Default implementations for CDPruner-related methods
+ov::Tensor InputsEmbedder::IInputsEmbedder::extract_text_features_for_pruning(const ov::Tensor& text_embeds,
+                                                                              const ov::Tensor& input_ids,
+                                                                              int64_t vision_start_token_id,
+                                                                              int64_t vision_end_token_id) const {
+    // Default implementation: return empty tensor
+    // Models that support CDPruner should override this method
```

Comment:
Is there a common implementation can fit most of VLM models? If yes, add the implementation here, if not, why not throw exception here?